### PR TITLE
OpenDAP router

### PR DESCRIPTION
**Pull request by abkfenris**
An initial pass at creating an OpenDAP router. It's not happy with datetime dimensions yet.

Works on #20

### CODE REVIEW

**Comment by mpiannucci** [comment]
Well this is awesome

File: environment.yml [hunk 1]
```diff
@@ -28,6 +28,7 @@ dependencies:
   - bokeh
   - pip:
       - git+https://github.com/xarray-contrib/xpublish.git@632a720aadba39cebaf062da7043835262d9fa3d
+      - opendap-protocol
```
Comment (ocefpaf) [inline environment.yml]: See https://github.com/conda-forge/staged-recipes/pull/18799

Comment (abkfenris) [reply environment.yml]: You clearly don't have enough to do.

**Comment by abkfenris** [comment]
And by using the encoded dtype rather than the interperted one, it seems to work, or at least give xarray enough to load a ds...haven't tried slicing around yet.

**Comment by abkfenris** [comment]
Time is not being encoded, but it's still, it's loading.

image

File: environment.yml (hunk 1 above)
Comment (ocefpaf) [reply environment.yml]: Look who is talking.

**Comment by abkfenris** [comment]
In the spirit of this repo being a playground, I'm gonna merge this and keep digging into the issues with other datasets.

**Comment by mpiannucci** [comment]
Yes!!! I think that's the right way to treat these PRs, open for posterity and visibility, but greedily merge to main

**Pull request merged**